### PR TITLE
add support for 1.16.1

### PR DESCRIPTION
**Pull request by Da-Dog**

### CODE REVIEW

**Comment by Fallen-Breath** [comment]
wait for 1.16 full release to come up and i'll merge

**Comment by Da-Dog** [comment]
1.16.1 Updated

File: MCDR-Bot.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 §7!!bot tp §b<name>§r：让名称为§b<name>§r的bot传送到你的位置
 §7!!bot clean§r：使所有bot离开游戏
 '''.strip()
-GAMEMODE = 'survival'
+GAMEMODE = 'creative'
```
Comment (Fallen-Breath) [inline MCDR-Bot.py]: why u changed this

Comment (Da-Dog) [reply MCDR-Bot.py]: Prevent People killing it when testing, ill change it back

**Comment by Fallen-Breath** [comment]
already done in v1.1.0

**Pull request closed without merging**